corehq/apps/domain/calculations.py

```diff
@@ -332,6 +332,51 @@ def _all_domain_stats():
     }
 
 
+def _calced_props(dom, id, all_stats):
```

Comment:
Underscore prefix seems weird, since this is imported elsewhere, but it's patterned after `_all_domain_stats`.